src/ReqResAutomation/LoginSuccessfulTest.java

```diff
@@ -1,52 +1,42 @@
-  package ReqResAutomation;
-  import static io.restassured.RestAssured.given;
-  import Resources.BaseUrl;
-  import SerializationPojo.LoginPojo;
-  import io.restassured.builder.RequestSpecBuilder;
-  import io.restassured.path.json.JsonPath;
-  import io.restassured.response.Response;
-  import io.restassured.specification.RequestSpecification;
-  
-
-
-  public class LoginSuccessfulTest
-  {
-    public static void main(String[] args)
-    {
-		
-	RequestSpecification req_spec = new RequestSpecBuilder().setBaseUri(BaseUrl.baseUri()).build();	
-    LoginPojo login= new LoginPojo();
-    login.setEmail("eve.holt@reqres.in");
-    login.setPassword("cityslicka");
-    Response response= given().spec(req_spec)
-   .body(login)
-   .header("Content-Type","application/json")
-   .when().post("api/login")
-   .then().assertThat()
-   .statusCode(200).header("Server", "cloudflare")
-   .extract().response();
-    String responseString = response.asString();
-    System.out.println(responseString);
-    System.out.println(login.getEmail());
-	System.out.println(login.getPassword());
-
-    JsonPath js = new JsonPath(responseString);
-    js.getString("token");
-    String getToken = js.getString("token");
-    System.out.println(getToken);
-
-
+package ReqResAutomation;
+
+import static io.restassured.RestAssured.given;
+import Resources.BaseUrl;
+import SerializationPojo.LoginPojo;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+public class LoginSuccessfulTest {
```

Comment:
This should be LoginTest. The idea is to have one class per api and then different methods test different test cases on that api.